pages/challenge.js

```diff
@@ -0,0 +1,154 @@
+import axios from 'axios';
+import Section from '../components/_common_/Section/Section';
```

Comment:
Can you change all of these file paths to be in line with module resolver (except the style import)

Examples:
```
import Section from 'components/_common_/Section/Section';
import namesFile from 'static/operationcode_challenge/names.txt';
```